examples/plugin/settings.gradle

```diff
@@ -0,0 +1,8 @@
+rootProject.name = 'plugin-example'
```

Comment:
In general, I think I'd prefer to have one example project per plugin type. That would allow people to copy-and-paste the example and adapt it to their needs, as I believe people mostly only need one type of plugin. Also, that would test our API / implementation separation in a better way.

Reply:
But you can copy it and delete what you don't need. Having a multi-module project is more duplication and might actually make it harder to copy it when you only want to implement one type of plugin. Or did you think of multiple standalone projects?

Reply:
Yes, multiple stand-alone projects, one for each plugin type. And each plugin project should only depend on one library project, not on the `cli` project.